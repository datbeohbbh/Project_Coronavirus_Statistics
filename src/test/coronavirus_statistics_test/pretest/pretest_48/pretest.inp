50 100
q 1 34 317
? 6
? 4 1
q 1 44 149
q 1 23 15
q 1 24 441
q 1 13 419
? 7
q 1 40 114
? 4 4
q 1 32 303
q 1 17 483
? 4 1
q 3 40 87
? 4 4
q 1 48 337
q 1 26 22
q 1 20 3
? 7
? 7
q 1 4 162
? 7
? 7
? 4 2
q 2 44 138
? 6
q 1 41 274
? 4 3
q 1 37 312
? 4 10
? 6
q 1 28 337
? 6
q 1 10 117
? 6
q 3 41 252
q 2 34 316
q 1 48 330
q 3 28 328
q 1 39 148
? 4 14
q 3 13 220
q 2 48 545
q 3 26 9
q 1 30 180
q 3 13 112
q 2 41 15
q 1 43 372
? 4 14
q 3 26 8
q 2 40 5
q 3 39 140
? 4 9
? 4 18
? 4 1
? 4 1
q 3 32 240
q 2 32 10
? 7
q 3 4 48
? 6
? 4 10
q 2 43 116
? 4 10
q 3 23 9
q 3 48 67
q 2 32 14
q 1 39 88
q 1 35 316
q 2 20 1
? 4 5
q 2 30 113
? 4 17
q 1 37 420
? 4 16
? 4 19
q 2 43 106
? 7
q 3 24 9
q 2 28 1
? 6
q 3 30 24
? 4 20
q 2 40 6
? 4 17
? 6
? 7
? 4 17
? 4 14
? 4 13
? 4 17
? 6
? 6
? 7
? 4 4
? 4 8
? 4 8
? 4 5
? 4 20
? 4 17