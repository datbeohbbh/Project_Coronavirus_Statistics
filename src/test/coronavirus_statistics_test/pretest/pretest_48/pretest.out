Query ? 6 :
Currently, no recovery cases have been recorded

Query ? 4 1 :
The answer is the city with the index 34 
- Information of the city 34 -
-> Number of current patients : 317
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
Currently, no deaths have been recorded

Query ? 4 4 :
The answer is the city with the index 34 
- Information of the city 34 -
-> Number of current patients : 317
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 23 
- Information of the city 23 -
-> Number of current patients : 15
-> Has recovered : 0
-> Was dead : 0

Query ? 4 4 :
The answer is the city with the index 32 
- Information of the city 32 -
-> Number of current patients : 303
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 40
Quantity: 87 people

Query ? 7 :
The city with the highest death is city with the index 40
Quantity: 87 people

Query ? 7 :
The city with the highest death is city with the index 40
Quantity: 87 people

Query ? 7 :
The city with the highest death is city with the index 40
Quantity: 87 people

Query ? 4 2 :
The answer is the city with the index 23 
- Information of the city 23 -
-> Number of current patients : 15
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 44
Quantity: 138 people

Query ? 4 3 :
The answer is the city with the index 23 
- Information of the city 23 -
-> Number of current patients : 15
-> Has recovered : 0
-> Was dead : 0

Query ? 4 10 :
The answer is the city with the index 34 
- Information of the city 34 -
-> Number of current patients : 317
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 44
Quantity: 138 people

Query ? 6 :
The city with the highest recovery is city with the index 44
Quantity: 138 people

Query ? 6 :
The city with the highest recovery is city with the index 44
Quantity: 138 people

Query ? 4 14 :
The answer is the city with the index 13 
- Information of the city 13 -
-> Number of current patients : 419
-> Has recovered : 0
-> Was dead : 0

Query ? 4 14 :
The answer is the city with the index 30 
- Information of the city 30 -
-> Number of current patients : 180
-> Has recovered : 0
-> Was dead : 0

Query ? 4 9 :
The answer is the city with the index 40 
- Information of the city 40 -
-> Number of current patients : 22
-> Has recovered : 5
-> Was dead : 87

Query ? 4 18 :
The answer is the city with the index 24 
- Information of the city 24 -
-> Number of current patients : 441
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 34 
- Information of the city 34 -
-> Number of current patients : 1
-> Has recovered : 316
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 34 
- Information of the city 34 -
-> Number of current patients : 1
-> Has recovered : 316
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 13
Quantity: 332 people

Query ? 6 :
The city with the highest recovery is city with the index 48
Quantity: 545 people

Query ? 4 10 :
The answer is the city with the index 32 
- Information of the city 32 -
-> Number of current patients : 53
-> Has recovered : 10
-> Was dead : 240

Query ? 4 10 :
The answer is the city with the index 32 
- Information of the city 32 -
-> Number of current patients : 53
-> Has recovered : 10
-> Was dead : 240

Query ? 4 5 :
The answer is the city with the index 41 
- Information of the city 41 -
-> Number of current patients : 7
-> Has recovered : 15
-> Was dead : 252

Query ? 4 17 :
The answer is the city with the index 37 
- Information of the city 37 -
-> Number of current patients : 312
-> Has recovered : 0
-> Was dead : 0

Query ? 4 16 :
The answer is the city with the index 43 
- Information of the city 43 -
-> Number of current patients : 256
-> Has recovered : 116
-> Was dead : 0

Query ? 4 19 :
The answer is the city with the index 17 
- Information of the city 17 -
-> Number of current patients : 483
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 13
Quantity: 332 people

Query ? 6 :
The city with the highest recovery is city with the index 48
Quantity: 545 people

Query ? 4 20 :
The answer is the city with the index 37 
- Information of the city 37 -
-> Number of current patients : 732
-> Has recovered : 0
-> Was dead : 0

Query ? 4 17 :
The answer is the city with the index 35 
- Information of the city 35 -
-> Number of current patients : 316
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 48
Quantity: 545 people

Query ? 7 :
The city with the highest death is city with the index 13
Quantity: 332 people

Query ? 4 17 :
The answer is the city with the index 35 
- Information of the city 35 -
-> Number of current patients : 316
-> Has recovered : 0
-> Was dead : 0

Query ? 4 14 :
The answer is the city with the index 4 
- Information of the city 4 -
-> Number of current patients : 114
-> Has recovered : 0
-> Was dead : 48

Query ? 4 13 :
The answer is the city with the index 39 
- Information of the city 39 -
-> Number of current patients : 96
-> Has recovered : 0
-> Was dead : 140

Query ? 4 17 :
The answer is the city with the index 35 
- Information of the city 35 -
-> Number of current patients : 316
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 48
Quantity: 545 people

Query ? 6 :
The city with the highest recovery is city with the index 48
Quantity: 545 people

Query ? 7 :
The city with the highest death is city with the index 13
Quantity: 332 people

Query ? 4 4 :
The answer is the city with the index 23 
- Information of the city 23 -
-> Number of current patients : 6
-> Has recovered : 0
-> Was dead : 9

Query ? 4 8 :
The answer is the city with the index 40 
- Information of the city 40 -
-> Number of current patients : 16
-> Has recovered : 11
-> Was dead : 87

Query ? 4 8 :
The answer is the city with the index 40 
- Information of the city 40 -
-> Number of current patients : 16
-> Has recovered : 11
-> Was dead : 87

Query ? 4 5 :
The answer is the city with the index 41 
- Information of the city 41 -
-> Number of current patients : 7
-> Has recovered : 15
-> Was dead : 252

Query ? 4 20 :
The answer is the city with the index 37 
- Information of the city 37 -
-> Number of current patients : 732
-> Has recovered : 0
-> Was dead : 0

Query ? 4 17 :
The answer is the city with the index 35 
- Information of the city 35 -
-> Number of current patients : 316
-> Has recovered : 0
-> Was dead : 0

